src/Microsoft.CodeAnalysis.Razor.Workspaces/ElementCompletionContext.cs

```diff
@@ -15,6 +15,7 @@ public sealed class ElementCompletionContext
             string containingTagName,
             IEnumerable<KeyValuePair<string, string>> attributes,
             string containingParentTagName,
+            bool containingParentIsTagHelper,
```

Comment:
We didn't ship this and VS doesn't use this type yet, it uses the one in `LanguageServices.Razor`.